test/app/components/path_container_component.html.erb

```diff
@@ -1 +1 @@
-<%= render PathComponent %>
\ No newline at end of file
+<%= render PathComponent.new %>
```

Comment:
Nitpick: I know it's not related to your change, but maybe we would want to add an empty newline here at the end? for the sake of consistency and also for having unix friendly files 🙈 what do you think?